backend/s390x/emit.mlp

```diff
@@ -546,6 +548,7 @@ let emit_instr i =
         let instr = name_for_specific sop in
         `	{emit_string instr}	{emit_reg i.res.(0)}, {emit_reg i.arg.(0)}, {emit_reg i.arg.(1)}\n`
     | Lop (Iname_for_debugger _) -> ()
+    | Lop (Iprobe _ |Iprobe_is_enabled _) -> assert false
```

Comment:
These should be `Misc.fatal_error "Not implemented"`, not `assert false`, I think?  The latter should be reserved for cases that are impossible to reach by inspection of nearby code, which isn't the case here.

Reply:
fixed.